lib/pattern-importer.js

```diff
@@ -0,0 +1,155 @@
+var through = require('through2'),
+  utility = require('./utility'),
+  merge = require('lodash.merge'),
+  gutil = require('gulp-util'),
+  path = require('path'),
+  fs = require('fs'),
+  PluginError = gutil.PluginError,
+  PLUGIN_NAME = 'pattern-importer';
+
+/*
+ * Imports html patterns that are following the standards at github.com/pattern-library
+ *
+ * @param {Object} options
+*/
+exports.patternImporter = function (options) {
```

Comment:
@scottnath I put this into a mock unit test and got this:
     Error opening category title-conversion data file: ./node_modules/pattern-library-utilities/lib/data/pattern-lab-categories.yml
 Error: Error: ENOENT, no such file or directory './node_modules/pattern-library-utilities/lib/data/pattern-lab-categories.yml'